frontend/src/utils/errorUtils.ts

```diff
@@ -0,0 +1,12 @@
+import { getLogger } from "@/helpers/logger";
+
+const logger = getLogger("api");
```

Comment:
I think @c0j0s has updated this to `wrapper` for his latest changes, and there is no specific `api`  type logger anymore, see [here](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g05/blob/master/frontend/src/helpers/logger.ts#L8). I think we should follow his way since we are going to standardise the way we create and use all the `api_wrappers` soon.